test/testenv/verificationutils.go

```diff
@@ -676,3 +676,22 @@ func VerifyAppsDownloadedByInitContainer(deployment *Deployment, testenvInstance
 		}
 	}
 }
+
+// VerifyBundlePush verify that bundle push was pushed on all indexers
+func VerifyBundlePush(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, replicas int, previousBundleHash string) {
+	gomega.Eventually(func() bool {
+		// Get Bundle status and check that each pod has successfully deployed the latest bundle
+		bundleStatus, err := GetBundlePushStatus(deployment, podName, ns, previousBundleHash)
+		gomega.Expect(err).To(gomega.Succeed(), "Unable to get bundle status", "Pod", podName)
+		if len(bundleStatus) < replicas {
+			return false
+		}
+		for pod, status := range bundleStatus {
+			if status != "Up" {
```

Comment:
Does this check for all pods in the deployment are up? Or is it for all pods specified in the `bundleStatus`?  Just thinking if theres a case when one indexer is slow to get the push.  Though now that i read this more, this is likely covered by the `len(bundleStatus)` above....

Reply:
Actually i was using the len to assert that its all the pods but i think i will update method to send the actual pod names. Might be better